test/plugins/collections.test.js

```diff
@@ -289,4 +290,98 @@ describe('collection plugin test', () => {
             });
         });
     });
+
+    describe('PUT /collections/{id}', () => {
+        const id = testCollection.id;
+        let options;
+        let updatedCollectionMock;
+
+        beforeEach(() => {
+            options = {
+                method: 'PUT',
+                url: `/collections/${id}`,
+                payload: {
+                    name: 'updated name',
+                    description: 'updated description',
+                    pipelineIds: [123, 124]
+                },
+                credentials: {
+                    username,
+                    scope: ['user']
+                }
+            };
+
+            updatedCollectionMock = hoek.clone(collectionMock);
+            updatedCollectionMock.name = 'updated name';
+            updatedCollectionMock.description = 'updated description';
+            updatedCollectionMock.pipelineIds = [123, 124];
+
+            collectionFactoryMock.get.withArgs({ id }).resolves(collectionMock);
+            collectionMock.update.resolves(updatedCollectionMock);
+            collectionMock.toJson.returns(updatedCollection);
```

Comment:
`updatedCollection` is from a file under `data` and it's only used at line 326. Can we get rid of the file and do it this way instead? https://github.com/screwdriver-cd/screwdriver/blob/master/test/plugins/tokens.test.js#L286